lib/jmespath/nodes/condition.rb

```diff
@@ -28,8 +28,7 @@ def optimize
 
     class ComparatorCondition < Node
       COMPARATOR_TO_CONDITION = {}
-      COMPARABLE_TYPES = [Integer, String].freeze
-
+      COMPARABLE_TYPES = [Integer, Float, String].freeze
```

Comment:
I noticed that after a constant there is always a new line in this file, except here after your change. But not sure if this required, as I don't see any style guides or Rubocop here.

Reply:
fixed

Reply:
Should Integer and Float instead be Numeric here?

Reply:
Using `Numeric` here instead of Integer and Float seems reasonable.